src/lib/route-data/route-data-registry.ts

```diff
@@ -26,8 +26,15 @@ export class RouteDataRegistry {
     }
 
     public static getAll(): Readonly<RouteData> {
```

Comment:
Der Type des Rückgabewertes stimmt noch nicht
```suggestion
    public static getAll(): { [path: string]: Readonly<RouteDataInterface> } {
```